### ISSUE TITLE
Schema generation fails for top level array

### ISSUE THREAD

**Issue by mrspacemankey** [coveooss/json-schema-for-humans#12]
First of all, thanks for this great tool! It generates really nice documentation.

One thing I could not get to work is generate the correct HTML for schemas with top level array definitions, e.g. see below:

```json
{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "title": "Example",
    "description": "Example for demonstration",
    "type": "array",
    "items": {
        "description": "A string",
        "type": "string"
    }
}
```

This should be valid json schema, but it generates the following output:
```html
<!DOCTYPE html><html lang=en> <head><link rel=stylesheet type=text/css href="https://fonts.googleapis.com/css?family=Overpass:300,400,600,800"><script src=https://code.jquery.com/jquery-3.4.1.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script><link href=https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css rel=stylesheet integrity=sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T crossorigin=anonymous><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js integrity=sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM crossorigin=anonymous></script><link rel=stylesheet type=text/css href=schema_doc.css><script src=https://use.fontawesome.com/facf9fa52c.js></script><title>Example</title><h1>Example</h1></head> <body></body> </html>
```

Which is just an empty template.

Is this case planned to be supported? If so, let me know if you need help implementing it.

Cheers!

**Comment by dblanchette** [comment]
Hi! This use case was not needed for us, so we neglected it. It seems pretty easy to implement though. I will take a look.

**Comment by dblanchette** [comment]
Fix released in version 0.5
